draft-irtf-cfrg-frost.md

```diff
@@ -845,19 +845,19 @@ The procedure for splitting a secret into shares is as follows.
   Errors:
   - "invalid parameters", if t > n
 
-  def secret_share(s, n, t):
+  def secret_share_split(s, n, t):
```

Comment:
```suggestion
  def secret_share_shard(s, n, t):
```

Can we also update the Sage code to match? The "shard" terminology comes from the [VDAF spec](https://cjpatton.github.io/vdaf/draft-patton-cfrg-vdaf.html#name-sharding), which I quite like.

Reply:
and thanks for linking `VDAFs`, in my list of TODO to read :)